msg/setup.py

```diff
@@ -0,0 +1,23 @@
+from setuptools import setup, find_packages
+
+
+__author__ = "Leonid Shamis"
+__copyright__ = "2022, Meta"
+
+
+install_requires = [
+    "pycapnp",
+    "mrp",
```

Comment:
This mrp depenency may not be necessary, as this disqualified installation on OSX